.github/workflows/ci.yml

```diff
@@ -229,10 +229,10 @@ jobs:
         include:
           - type: "qiskit-superstaq"
             pattern: "*_qss.ipynb qiskit-superstaq"
-            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq -e ./supermarq-benchmarks"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq"
           - type: "cirq-superstaq"
             pattern: "*_css.ipynb cirq-superstaq"
-            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq -e ./cirq-superstaq -e ./supermarq-benchmarks"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq -e ./cirq-superstaq[examples]"
```

Comment:
Is the idea to introduce a seperate check with supermarq in light of new (tangentially, see [discussion](https://github.com/Infleqtion/server-superstaq/discussions/3352#discussioncomment-10463631)) QCVV notebooks

Reply:
(outside the scope of this PR, but while on the topic of notebook checks, is https://github.com/Infleqtion/client-superstaq/issues/752 now stale?)

Reply:
yeah was gonna ask you about this - right now the supermarq notebooks (and dd notebook) are excluded from tests: https://github.com/Infleqtion/client-superstaq/blob/8f68a759b743c159dadfa93c0f0613bcc1b83304/checks/pytest_.py#L10-L11

the dd notebook definitely should be excluded (it submits to hardware, and needs a rewrite anyway), but i wasn't sure if the other exclusions were intentional/necessary?

if we add them back in, i do think i'd prefer to make it a third "supermarq-benchmarks" type here though, instead of including them in the css/qss ones (both to make them finish faster, and so these double as validators of the example-requirements files)

Reply:
I'm not entirely sure what the rationale was for `args += ["--exclude", "docs/source/apps/*"]` (seems like added in https://github.com/Infleqtion/client-superstaq/pull/428), but my question was originally prompted from noticing that, for example, the `cirq-superstaq` notebook check doesn't include `supermarq-benchmarks/examples/Supermarq_HPCA_Tutorial_css.ipynb` -- however, I'm realizing now this has more to do with https://github.com/Infleqtion/client-superstaq/pull/1031 and the aforementioned exclusion haha. 

I'm on the side that we probably should create a separate 'supermarq-benchmarks` notebook check if we plan to add new notebooks and the checks can pinpoint incompatibility with new changes (if they don't take _too_ long). However, happy to leave this to a separate PR once those new notebooks are consolidated and we can check runtime or something.

Reply:
ok, added a supermarq notebook check to the matrix, currently just covering the notebooks that seemed to be excluded accidentally